pkg/liqonet/doc.go

```diff
@@ -1,2 +0,0 @@
-// Package liqonet all the related functions for the liqo network are contained in this package
```

Comment:
Should it be really removed?

Reply:
After this PR, directory liqonet contains only directories(which are packages) and 0 files. An external file would import a specific liqonet package, instead of liqonet.